tests/converter/dialogflow.py

```diff
@@ -0,0 +1,33 @@
+"""This module tests the convertion tool that is used to convert a rasa or diaglogflow
+project into a mindmeld project"""
+
+from mindmeld import NaturalLanguageProcessor
+from mindmeld.converter.dialogflow import DialogFlowConverter
+
+
+def test_dialogflow_converter(dialogflow_project_path, mindmeld_project_path2):
+    converter = DialogFlowConverter(dialogflow_project_path, mindmeld_project_path2)
+    converter.convert_project()
+
+def test_nlp_greet(mindmeld_project_path2):
+    nlp = NaturalLanguageProcessor(mindmeld_project_path2)
```

Comment:
Try to use fixture so that you do not have to create and build different NLP files - check the Rasa test files